src/mlio-py/mlio/contrib/insights/column_statistics.h

```diff
@@ -45,13 +45,26 @@ class Column_analysis {
         return static_cast<std::size_t>(std::round(str_cardinality_estimator_.estimate()));
     }
 
+public:
+    double estimate_median_approx() const
+    {
+        if (numeric_column_sample.empty()) {
+            return std::nan("");
+        }
+        std::vector<double> v = numeric_column_sample;
```

Comment:
In case `numeric_column_sample` can be private, there is no need to copy it over to another variable.

Reply:
Will change to `mutable`